src/output-json-http.c

```diff
@@ -294,19 +295,12 @@ static void EveHttpLogJSONExtended(JsonBuilder *js, htp_tx_t *tx)
     const int resp = tx->response_status_number;
     if (resp > 0) {
         jb_set_uint(js, "status", (uint32_t)resp);
-    } else if (tx->response_status != NULL) {
```

Comment:
This belongs in it's own commit

Reply:
and own issue and SV test with https://redmine.openinfosecfoundation.org/issues/7311